WTWSMS/common/scripted_triggers/ACR_scripted_triggers.txt

```diff
@@ -83,6 +88,7 @@ is_ancient_religion_trigger = {
 		religion = hellenic_pagan
 		religion = egyptian_pagan
 		religion = aten_pagan
+		religion = assyrian_pagan
```

Comment:
It doesn't have to be strictly used for ACR. It can and should be expanded to other ancient religions.